generators/dockertools/templates/python/Dockerfile-tools

```diff
@@ -14,13 +14,13 @@ RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-
 <% if (language === "PYTHON") { %>
 ENV FLASK_APP=server/__init__.py
 ENV FLASK_DEBUG=true
+
+
+<% } %>
 COPY run-dev /bin
 RUN chmod 777 /bin/run-dev
 RUN /bin/run-dev
```

Comment:
I don't think we want to run the command during the build. The running should be reserved for `idt shell run-dev` or `bx dev shell run-dev`